tools/test-ci.sh

```diff
@@ -0,0 +1,3 @@
+pushd `dirname $0`/.. > /dev/null
+`which python` tools/test.py -J --mode=release $*
```

Comment:
Regarding your question in the PR description, you may want to look into using $PYTHON here (and possible exporting that in the Makefile beforehand, I’m not sure)